File: python/Bioskop.py
class Film:
    def __init__(self, id:int,poster:str, judul:str, genre:str, durasi:int, harga:int, sutradara:str):
        self._id = int(id)
        self._poster = str(poster)
        self._judul = str(judul)
        self._genre = str(genre)
        self._durasi = int(durasi)
        self._harga = int(harga)
        self._sutradara = str(sutradara)

    # ---- Getter -----    
    def get_id(self) -> int:
        return self._id
    def get_poster(self) -> str:
        return self._poster
    def get_judul(self) -> str:
        return self._judul    
    def get_genre(self) -> str:
        return self._genre
    def get_durasi(self) -> int:
        return self._durasi
    def get_harga(self) -> int:
        return self._harga
    def get_sutradara(self) -> str:
        return self._sutradara

    # ---- Setter ----aa
    def set_id(self, id: int) -> None:
        self._id = int(id)
    def set_poster(self, poster: str) -> None:
        self._poster = str(poster)
    def set_judul(self, judul: str) -> None:
        self._judul = str(judul)
    def set_genre(self, genre: str) -> None:
        self._genre = str(genre)
    def set_durasi(self, durasi: int) -> None:
        self._durasi = int(durasi)
    def set_harga(self, harga: int) -> None:
        self._harga = int(harga)
    def set_sutradara(self, sutradara: str) -> None:
        self._sutradara = str(sutradara)

File: php/Bioskop.php
<?php

class Film {
    private $id;
    private $poster;
    private $judul;
    private $genre;
    private $durasi;
    private $harga;
    private $sutradara;

    // Constructor
    public function __construct($id, $poster, $judul, $genre, $durasi, $harga, $sutradara) {
        $this->id = (int) $id;
        $this->poster = (string) $poster;
        $this->judul = (string) $judul;
        $this->genre = (string) $genre;
        $this->durasi = (int) $durasi;
        $this->harga = (int) $harga;
        $this->sutradara = (string) $sutradara;
    }

    // ---- Getter ----
    public function getId() { return $this->id; }
    public function getPoster() { return $this->poster; }
    public function getJudul() { return $this->judul; }
    public function getGenre() { return $this->genre; }
    public function getDurasi() { return $this->durasi; }
    public function getHarga() { return $this->harga; }
    public function getSutradara() { return $this->sutradara; }

    // ---- Setter ----
    public function setId($id) { $this->id = (int) $id; }
    public function setPoster($poster) { $this->poster = (string) $poster; }
    public function setJudul($judul) { $this->judul = (string) $judul; }
    public function setGenre($genre) { $this->genre = (string) $genre; }
    public function setDurasi($durasi) { $this->durasi = (int) $durasi; }
    public function setHarga($harga) { $this->harga = (int) $harga; }
    public function setSutradara($sutradara) { $this->sutradara = (string) $sutradara; }
}

